Gems/Atom/Tools/AtomToolsFramework/Code/Source/PreviewRenderer/PreviewRenderer.cpp

```diff
@@ -79,6 +79,9 @@ namespace AtomToolsFramework
         m_view->SetViewToClipMatrix(viewToClipMatrix);
         m_renderPipeline->SetDefaultView(m_view);
 
+        // Don't render the pipeline until a capture request
+        m_renderPipeline->RemoveFromRenderTick();
+
```

Comment:
I tried this when I updated the frame capture component and it broke all previews for me, please validate previews still work.